src/sql/parts/profiler/editor/profilerInput.ts

```diff
@@ -123,7 +125,20 @@ export class ProfilerInput extends EditorInput implements IProfilerSession {
 		return this._state;
 	}
 
+	public onSessionStopped(notification: sqlops.ProfilerSessionStoppedParams) {
+		this._notificationService.error(nls.localize("profiler.sessionStopped", "Session stopped unexpectedly on the server."));
```

Comment:
I would probably change this to "XEvent Profiler Session stopped unexpectedly on the server {connection_servername_here}."  For reasons similar to outline in other comment.